File: programacao/c/respostas/selecao/26.c
#include <stdio.h>

int main(void) {
    float dist, consumog, consumoe, precog, precoe, custog, custoe;

    printf("Distancia a percorrer (km): ");
    scanf("%f", &dist);
    printf("Consumo com gasolina (km/l): ");
    scanf("%f", &consumog);
    printf("Consumo com etanol (km/l): ");
    scanf("%f", &consumoe);
    printf("Preco da gasolina (R$): ");
    scanf("%f", &precog);
    printf("Preco do etanol (R$): ");
    scanf("%f", &precoe);

    custog = (dist / consumog) * precog;
    custoe = (dist / consumoe) * precoe;

    if (custog < custoe) {
        printf("Gasolina e mais vantajosa\n");
        printf("Custo: %.2f\n", custog);
    } else {
        printf("Etanol e mais vantajoso\n");
        printf("Custo: %.2f\n", custoe);
    }

    return 0;
}

File: programacao/python/respostas/selecao/26.py
dist = float(input("Distancia a percorrer (km): "))
consumog = float(input("Consumo com gasolina (km/l): "))
consumoe = float(input("Consumo com etanol (km/l): "))
precog = float(input("Preco da gasolina (R$): "))
precoe = float(input("Preco do etanol (R$): "))

custog = (dist / consumog) * precog
custoe = (dist / consumoe) * precoe

if custog < custoe:
    print("Gasolina e mais vantajosa")
    print(f"Custo: {custog:.2f}")
else:
    print("Etanol e mais vantajoso")
    print(f"Custo: {custoe:.2f}")
